SpeedyTransforms/Project.toml

```diff
@@ -31,7 +31,7 @@ RingGrids = {path = "../RingGrids"}
 AbstractFFTs = "1.5.0"
 Adapt = "4.3.0"
 AssociatedLegendrePolynomials = "1.0.1"
-Atomix = "1.1.2"
+Atomix = "1"
```

Comment:
Why was this specific before? just checking what would happen if someone ended up installing a version between [1, 1.1.2)